schemas/client.yml

```
@@ -48,9 +48,15 @@
     - name: mac
       level: core
       type: keyword
+      example: 00-00-5E-00-53-23
       description: >
         MAC address of the client.
 
+        The format from RFC 7042 is used: Each octet (that is, 8-bit byte) is
```

```suggestion
        The notation format from RFC 7042 is suggested: Each octet (that is, 8-bit byte) is
```

The description looks great. My only suggestion is to soften the wording to remove any impression that ECS is mandating any one format as of this time.